web/libs/editor/src/components/HtxTextBox/HtxTextBox.jsx

```diff
@@ -178,12 +182,29 @@ export class HtxTextBox extends React.Component {
       <>
         <Paragraph {...props}>
           <span ref={this.textRef}>{text}</span>
-          {isEditable && onChange && (
-            <EditOutlined onClick={this.startEditing} aria-label="Edit Region" className="ant-typography-edit" />
-          )}
         </Paragraph>
+        {isEditable && onChange && (
+          <Button
+            type="text"
+            className={styles.button}
+            tooltip="Edit"
+            tooltipTheme="Dark"
+            style={{ padding: 0 }}
+            icon={<IconEdit />}
+            onClick={this.startEditing}
+          />
+        )}
         {isDeleteable && onDelete && (
-          <DeleteOutlined className={styles.delete} aria-label="Delete Region" onClick={onDelete} />
+          <Button
+            type="text"
```

Comment:
ahh e2e is failing because its looking for `aria-label="Delete Region"` that was there before. can you add that back in?